### PR TITLE
update onboarding account

### PR DESCRIPTION
**Pull request by jtomchak**
Update to onboarding follow suggestions

### CODE REVIEW

File: app/lib/onboarding/v2/onboarding_categories_production.yml [hunk 1]
```diff
@@ -76,9 +70,12 @@
...
       type: account
       summary: Award-winning law journalist.
-    - account: '@Emilyalpertreyes@mas.to'
+    - account: '@404mediaco@mastodon.social'
+      type: account
+      summary:
+    - account: '@egies@masto.ai'
       type: account
-      summary: LA Times covering Public Health.
+      summary:
```
Comment (rileyhCode) [inline app/lib/onboarding/v2/onboarding_categories_production.yml:78]: No summary for this account?

File: app/lib/onboarding/v2/onboarding_categories_production.yml [hunk 2]
```diff
@@ -122,9 +116,9 @@
     - account: '@caseynewton@mastodon.social'
       type: account
       summary: Tech journalist. Editor, Platformer.
-    - account: '@DetroitBORG@techhub.social'
+    - account: '@carnage4life@mas.to'
       type: account
-      summary: Popular Apple and tech YouTuber.
+      summary:
```
Comment (rileyhCode) [inline app/lib/onboarding/v2/onboarding_categories_production.yml:121]: Missing summary?

File: app/lib/onboarding/v2/onboarding_categories_production.yml [hunk 3]
```diff
@@ -160,21 +148,21 @@
     - account: '@absolutspacegrl@mastodon.social'
       type: account
       summary: NASA engineer.
+    - account: '@mariapopova@indieweb.social'
+      type: account
+      summary:
```
Comment (rileyhCode) [inline app/lib/onboarding/v2/onboarding_categories_production.yml:153]: missing summary

File: app/lib/onboarding/v2/onboarding_categories_production.yml [hunk 4]
```diff
@@ -160,21 +148,21 @@
...
+    - account: '@luckytran@med-mastodon.com'
+      type: account
+      summary:
```
Comment (rileyhCode) [inline app/lib/onboarding/v2/onboarding_categories_production.yml:156]: missing summary

**Comment by jtomchak** [comment]
Summary is no longer being used as client is loading profile info

RocketSim_Screenshot_iPhone_14_Pro_2023-12-03_17 13 44

**Pull request merged**